src/SymbolCollector.Android.Library/Host.cs

```diff
@@ -137,13 +139,13 @@ public static IHost Init(string dsn)
         }
     }
 
-    public class AndroidClientHandlerBuilder : HttpMessageHandlerBuilder
+    public class AndroidMessageHandlerBuilder : HttpMessageHandlerBuilder
```

Comment:
This was done to use `AndroidClientHandler` when the LetsEncrypt root cert thing expired and broken Xamarin.Android. Now that we're back (`AndroidClientHandler` warned to be obsolete) we might be able to get rid of this altogether.

Lets see the result of the test run in the device farm (success rate)